client/src/components/Snackbar/index.jsx

```diff
@@ -46,9 +54,14 @@ const useStyles1 = makeStyles((theme) => ({
   },
 }));
 
-function MySnackbarContentWrapper(props) {
+const MySnackbarContentWrapper = ({
+  className,
+  message,
+  onClose,
+  variant,
+  ...other
+}) => {
   const classes = useStyles1();
```

Comment:
there is inconsistency in styling.. erm, style.. across different parts of the front end. We should probably fix this in the last week.

Reply:
Agreed, issue #36 added.